Improve site accessibility score

SDBrett opened the pull request:
Fixes: #1016

Updates to site colours to align with VMware branding and improve contrast rations for site accessibility.

Update to youtube plugin for that it can will insert an iframes title as well. New usage of liquid template `{% youtube "<title>" %}

Updates to links to provide link text

Updates to images to add alt text.

Accessibility changes assist people visiting the site with visual impairments and improve the function of text to speech tools such as Jaws.

Change includes moving custom-site/css/sonobouy.css to /site/_scss/settings/ for configuration consistency.

Signed-off-by: Brett Johnson <brett@sdbrett.com>


**What this PR does / why we need it**:

**Which issue(s) this PR fixes**
- Fixes #

**Special notes for your reviewer**:

**Release note**:
```
release-note
```

johnSchnake commented on site/docs/master/README.md:
```diff
@@ -1,4 +1,4 @@
-# <img src="img/sonobuoy-logo.png" width="400px" > [![CircleCI](https://circleci.com/gh/vmware-tanzu/sonobuoy.svg?style=svg)](https://circleci.com/gh/vmware-tanzu/sonobuoy)
+# <img src="img/sonobuoy-logo.png" alt="Sonobuoy logo" width="400px" > [![CircleCI](https://circleci.com/gh/vmware-tanzu/sonobuoy.svg?style=svg)](https://circleci.com/gh/vmware-tanzu/sonobuoy)
```
So CI is failing because you edited this file directly. In order to keep the github README.md in sync with this document we treat the root README.md as the source of truth and script the updating of this one.

```
scripts/sync_readme.sh modified the git status. If updating the README.md, update the root README.md and run that script.
```

SDBrett replied:
Thanks mate, 

I've updated the root README.md and run the sync script.

SDBrett commented:
Made a minor change to improve link colour in text based on feedback that they did not stand out enough and where easy to miss. 

Added link colour for section.grey as lighter links do not have enough contrast ratio with the grey background. This is used on the home page.

Rebased

johnSchnake commented:
Thanks a ton for the help with this submission.
